<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoDialog</class>
 <widget class="QDialog" name="VideoDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Video Management</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>281</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Videos</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="toolTip">
           <string>Use this to set the distance of the camera from the reference point, where the etalon is.</string>
          </property>
          <property name="text">
           <string>Camera Distance:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="camDistBox">
          <property name="toolTip">
           <string>Use this to set the distance of the camera from the reference point, where the etalon is.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="dbListView">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="addVid">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Add new video</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1462036180_list-add.png</normaloff>:/VAM/Icons/1462036180_list-add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteVid">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Remove video</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1462036183_document-delete3.png</normaloff>:/VAM/Icons/1462036183_document-delete3.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vidUp">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Move video up</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1462036195_arrow-up.png</normaloff>:/VAM/Icons/1462036195_arrow-up.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="vidDown">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Move video down</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1462036198_arrow-down.png</normaloff>:/VAM/Icons/1462036198_arrow-down.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="saveBtn">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Save video list</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1466084211_Accept.png</normaloff>:/VAM/Icons/1466084211_Accept.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>30</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Return without saving</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset resource="VAM.qrc">
            <normaloff>:/VAM/Icons/1466084143_delete.png</normaloff>:/VAM/Icons/1466084143_delete.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="VAM.qrc"/>
 </resources>
 <connections/>
</ui>
